### DISCUSSION TITLE
Could you create a mirror on my Gitea instance?

### DISCUSSION THREAD

**Discussion by hello-smile6** [Darin755/browser-linux#4] [General]
[My Gitea instance](https://gitea.9pfs.repl.co/)
You can [sign in with GitHub](https://gitea.9pfs.repl.co/user/oauth2/github), if you'd prefer that. I'd just like to have a mirror of this repo on my server.

**Comment by hello-smile6** [comment]
@Darin755 If you don't want to, I could create one, but I'd prefer to not have it on my account.

**Reply by Darin755** [reply]
You're welcome to create a separate mirror because this is licensed under the GPL but I will not be creating one due to it making things more complex. I hope this is not to big of a inconvence to you.
